client/rpcserver/handlers.go

```diff
@@ -812,8 +812,8 @@ var helpMsgs = map[string]helpMsg{
     assetID (int): The asset's BIP-44 registered coin index. e.g. 42 for DCR.
       See https://github.com/satoshilabs/slips/blob/master/slip-0044.md
     walletType (string): The wallet type.
-     path (string): Optional. The path to a configuration file.
-     settings (string): A JSON-encoded string->string mapping of additional
+    path (string): Optional. The path to a configuration file.
+    settings (string): A JSON-encoded string->string mapping of additional
```

Comment:
Looks like formatting here was slightly off: (took liberty to fix it, since I noticed)

<img width="1344" alt="image" src="https://user-images.githubusercontent.com/7031569/178561821-29d2c783-6163-48b3-bcc4-401b90999955.png">